app/renderer/components/main/braveryPanel.js

```diff
@@ -184,164 +240,692 @@ class BraveryPanel extends ImmutableComponent {
-                  : null
-                }
-                {
-                  this.isBlockingTrackedContent
-                  ? this.blockedByTrackingList.map((site) => <li key={site}>{site}</li>)
-                  : null
-                }
-              </ul></li>
-              : null
-            }
-            {
-              this.isRedirectingResources && this.isHttpseShown
-              ? <li><ul>
-                {
-                  this.redirectedResourcesSet.map((site) =>
-                    <li key={site}>{site}</li>)
-                }
-              </ul></li>
-              : null
-            }
-            {
-              this.isBlockingScripts && this.isBlockedScriptsShown
-              ? <li><ul>
-                {
-                  this.blockedScripts.map((site) =>
-                    <li key={site}>{site}</li>)
-                }
-              </ul></li>
-              : null
-            }
-            {
-              this.isBlockingFingerprinting && this.isFpShown
-              ? <li><ul>
-                {
-                  this.blockedFingerprinting.map((site) =>
-                    <li key={site}>{site}</li>)
-                }
-              </ul></li>
-              : null
-            }
-          </ul>
-          <div className={cx({
-            braveryAdvancedTitle: true,
-            disabled: !shieldsUp
-          })} onClick={this.onToggleAdvanced}>
+    const compactBraveryPanel = getSetting(settings.COMPACT_BRAVERY_PANEL)
+
+    return <Dialog onHide={this.props.onHide} testId='braveryPanelContainer' isClickDismiss>
+      <div className={css(
+        commonStyles.flyoutDialog,
+        styles.braveryPanel,
+        compactBraveryPanel && styles.braveryPanel_compact
+      )}
+        onClick={(e) => e.stopPropagation()}
+        data-test-id={compactBraveryPanel ? 'braveryPanelCompact' : 'braveryPanel'}>
+        {
+          compactBraveryPanel
+          ? this.compactBraveryPanelHeader
+          : this.defaultBraveryPanelHeader
+        }
+        <section className={css(
+          styles.braveryPanel__stats,
+          compactBraveryPanel && styles.braveryPanel_compact__stats
+        )}>
+          <div data-test-id='adsBlockedStat'
+            onClick={this.onToggleAdsAndTracking}
+            className={cx({
+              [css(styles.braveryPanel__stats__item_count_clickable)]: !!adsBlockedStat,
+              [css(styles.braveryPanel__stats__item_count_disabled)]: !shieldsUp || adControl === 'allowAdsAndTracking',
+              [css(gridStyles.row1col1)]: !compactBraveryPanel,
+              [css(gridStyles.row1col1)]: compactBraveryPanel,
+              [css(styles.braveryPanel__stats__item_count_adsBlockedStat)]: true,
+              [css(styles.braveryPanel__stats__item, styles.braveryPanel__stats__item_count)]: !compactBraveryPanel,
+              [css(styles.braveryPanel_compact__stats__item_count)]: compactBraveryPanel
```

Comment:
since all styles use Aphrodite we could make use of its recommended syntax as well:

```js
className={css(
  !!adsBlockedStat && styles.braveryPanel__stats__item_count_clickable,
  (!shieldsUp || adControl === 'allowAdsAndTracking') && styles.braveryPanel__stats__item_count_disabled,
  gridStyles.row1col1,
  styles.braveryPanel__stats__item_count_adsBlockedStat,
  !compactBraveryPanel && styles.braveryPanel__stats__item,
  !compactBraveryPanel && styles.braveryPanel__stats__item_count,
  compactBraveryPanel && styles.braveryPanel_compact__stats__item_count 
)}
```

it may seem weird at first but it looks for the first assumption and only execute the latest if condition evaluates to true. `condition && style` would write nothing if condition is false.

Same tip applies for next cases. Let me know if you need a hand rewriting it